common/cassandra/cassandraCluster.go

```diff
@@ -70,6 +77,16 @@ func NewCassandraCluster(cfg config.Cassandra) *gocql.ClusterConfig {
 	return cluster
 }
 
+func RegionHostFilter() gocql.HostFilter {
```

Comment:
This leaks some proprietary implementation details into open-source. Should we instead use the Option pattern to allow the caller to provide a custom `gocql.HostFilter` which overrides the default?